src/app/story/storyrow/storyrow.component.css

```diff
@@ -142,6 +146,32 @@
     right: 0px;
 }
 
+.node-uid-column__display-name {
+    font-weight: bold;
+}
+
+.node-uid-column__timestamp {
+    font-size: 1.1rem;
+}
+
+.node-uid-column__clock-icon {
+    /* We want it to be on top of everything except the sidebar */
+    z-index: 97;
+    margin-left: 4.1rem;
+    bottom: 1.2rem;
+    position: absolute;
+    width: 12%;
```

Comment:
Instead of 12%, could you just set this to `22px`? This will make the size consistent. In this case since it's an image, pixels are just fine.